pkg/opcua/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:16.04
+
+RUN mkdir -p kubeedge ca
+
+COPY ./opcua kubeedge/
+COPY ./config.yaml kubeedge/
+COPY ./ca /ca
```

Comment:
It could be better if provide a tool to generate certs.

Reply:
Thanks for your comment. The cert is a demo. It depends on the external opcua server.

Reply:
I think we should provide a tool to generate them although it is a demo.

Reply:
We can remove it from dockerfile/image, and mount into contaier if needed

Reply:
The customer could package it without any file or with the files for them own. It depends on the authentication method the opcua server requires. This folder is an example that the customer could refer to.